services/checks/checks.go

```diff
@@ -131,6 +147,17 @@ func New(agentsRegistry agentsRegistry, alertsRegistry alertRegistry, db *reform
 		s.startDelay = 0
 	}
 
+	s.mScriptsExecuted.WithLabelValues(string(models.MySQLServiceType))
+	s.mScriptsExecuted.WithLabelValues(string(models.PostgreSQLServiceType))
+	s.mScriptsExecuted.WithLabelValues(string(models.MongoDBServiceType))
+	s.mAlertsGenerated.WithLabelValues(string(models.MySQLServiceType), string(check.MySQLShow))
```

Comment:
Let's insert a blank line there when we change metric

Reply:
done in this commit https://github.com/percona/pmm-managed/pull/407/commits/487a1ec497c921d94a561a2383252470bf3cb251  cc @AlekSi